sources/packages/backend/libs/integrations/src/sfas-integration/sfas-integration.processing.service.ts

```diff
@@ -141,10 +141,15 @@ export class SFASIntegrationProcessingService {
 
       if (result.success) {
         try {
+          /**
+           * Bulk operation to update student id in SFAS individuals table after importing data from SFAS.
+           */
+          await this.sfasIndividualService.updateStudentId();
           await this.sfasIndividualService.updateDisbursementOverawards();
+          await this.sfasRestrictionService.updateStudentRestrictions();
```

Comment:
The operation related `updateStudentId`, `updateDisbursementOverawards`, and `updateStudentRestrictions` can have their errors individually logged.
For instance, the below message is trying to comunicating 2 out of 3 possible issues (e.g. `updateStudentId` is not included).
"Error while updating overawards balances or inserting student restrictions imported from SFAS." 

Besides that, I believe that we should not be deleting the file if one of those operations failed. All imported data from SFAS should be able to safely retry. If I am not wrong we can execute the import of the same file over and over again with no harm.

For instance, the comment `Delete the file only if it was processed with success.` states that the file should be deleted if everything went well and I believe that we broke this rule when we did the overawards (unless I am missing something).

In summary, I believe that we should delete the file only when `updateStudentId`, `updateDisbursementOverawards`, and `updateStudentRestrictions` are successfully executed. In case one of them fail the file should be fixed and processed again.

@dheepak-aot @guru-aot @andrepestana-aot @lewischen-aot thoughts?

Reply:
@andrewsignori-aot  You're right about checking success before deleting the file and that would be good to have separate log messages for the operations.

Reply:
Yes even a singe operation failure should result in a retrying it by fixing again. I do agree. Also as @andrepestana-aot mentioned it would be better to have some log messages for these.